lib/l10n/app_mr.arb

```
@@ -0,0 +1,631 @@
+    "callDeclined": "कॉल नाकारला",
+    "callDeclinedTo": "वापरकर्ता ${username} ने कॉल नाकारला.",
+    "@callDeclinedTo": {
+        "description": "जेव्हा कॉल नाकारला जातो तेव्हा संदेश",
+        "placeholders": {
+            "username": {
+                "type": "String",
+                "example": "john_doe"
+            }
+        }
+    },
+    "checkForUpdates": "अपडेट तपासा",
```

RuboCop (1.80.2)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)